Automattic/jetpack
# E2E tests: disable tests on workflow_run event

#### Changes proposed in this Pull Request:

Remove the e2e tests run after build is completed.
Now with tests running on updates in mirror repos the run on `workflow_run` after build is completed is redundant.

#### Other information:


#### Jetpack product discussion
pd5faL-kN-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Check that after Build workflow completes when this is merged no e2e tests are running for `workflow_run` event
